Add a config option to specify custom command-line option

littlewu2508 opened the pull request:
Closes: https://github.com/34736384/genshin-fps-unlock/issues/476

littlewu2508 committed:
Add a config option to specify custom command-line option

Closes: https://github.com/34736384/genshin-fps-unlock/issues/476

littlewu2508 commented:
I have compiled it on Debian with the modification of https://github.com/34736384/genshin-fps-unlock/pull/478: 
[unlockfps_nc_custom_option.zip](https://github.com/user-attachments/files/17430678/unlockfps_nc.zip)

34736384 commented on unlockfps_nc/Model/Config.cs:
```diff
@@ -26,6 +26,7 @@ public class Config
         public int CustomResY { get; set; } = 1080;
         public int MonitorNum { get; set; } = 1;
         public int Priority { get; set; } = 3;
+        public string CustomOption { get; set; } = "";
```
```suggestion
        public string AdditionalCommandLine { get; set; } = "";
```
'CustomOption' seems to be ambiguous about what it does

34736384 commented on unlockfps_nc/Service/ProcessService.cs:
```diff
@@ -234,6 +234,7 @@ private string BuildCommandLine()
                 commandLine += "use_mobile_platform -is_cloud 1 -platform_type CLOUD_THIRD_PARTY_MOBILE ";
 
             commandLine += $"-monitor {_config.MonitorNum} ";
+            commandLine += $"{_config.CustomOption} ";
```
```suggestion
            commandLine += $"{_config.AdditionalCommandLine} ";
```

34736384 reviewed (changes requested):
Hello, please see the requested changes below

littlewu2508 committed:
Use  AdditionalCommandLine instead of CustomOption

'CustomOption' seems to be ambiguous about what it does.

The real purpose is to append additional command-line parameter

Co-authored-by: REL <34736384@users.noreply.github.com>
